sys/targets/targets.go

```diff
@@ -291,8 +291,11 @@ var List = map[string]map[string]*Target{
 			KernelHeaderArch: "x86",
 		},
 		ARM64: {
-			PtrSize:          8,
-			PageSize:         4 << 10,
+			PtrSize:  8,
+			PageSize: 4 << 10,
+			// Jump tables mess up with SyzOS execution, disable them for now.
+			// See https://github.com/google/syzkaller/issues/5565 for context.
+			CFlags:           []string{"-fno-jump-tables"},
```

Comment:
This won't for reproducers since we historically provided reproducers that worked with any flags.

Reply:
Right, I'll revert this.